src/javascript/binary/websocket_pages/user/lost_password.js

```diff
@@ -6,7 +6,7 @@ const LostPassword = (() => {
 
     const responseHandler = (response) => {
         if (response.verify_email) {
-            $(form_id).html($('<div/>', { class: 'notice-msg', text: localize('Please check your email to complete the process.') }));
+            $(form_id).html($('<div/>', { class: 'notice-msg', text: localize('Please check your email for the password reset link.') }));
```

Comment:
please update the text in GenerateStaticData.pm as well or translations won't be reflected

Reply:
ok, noted